src/views/MissionPlanningView.vue

```diff
@@ -769,6 +878,10 @@ const generateWaypointsFromSurvey = (): void => {
   showDialog({ variant: 'success', message: 'Waypoints generated from survey path.', timer: 1000 })
 }
 
+provide('surveyLinesAngle', (angle: number) => {
+  surveyLinesAngle.value = angle
+})
+
```

Comment:
This inject/provide implementation had been removed on a previous review, haven't it?

Any reason to use it here, since the ScanDirectionDial is a direct child of the MissionPlanningView?

Reply:
[discussed in private about it and came to the conclusion that it does not worth to use the provide/injection system here]